lib/web/apiserver.go

```diff
@@ -4500,6 +4500,7 @@ var authnWsUpgrader = websocket.Upgrader{
 	ReadBufferSize:  1024,
 	WriteBufferSize: 1024,
 	CheckOrigin:     func(r *http.Request) bool { return true },
+	Error:           func(w http.ResponseWriter, r *http.Request, status int, reason error) {},
```

Comment:
can you add a comment explaining why we disable the error handler here:

something like:

```suggestion
	// We’re disabling the error handler here since error handling is managed within the handler itself. 
	// Allowing the WS error handler to operate would result in it writing to the response writer, 
	// which conflicts with our custom error handler and leads to unintended errors.
	Error:           func(http.ResponseWriter, *http.Request, int, error) {},
```